core/src/main/java/edu/wpi/grip/core/sockets/SocketHints.java

```diff
@@ -193,5 +193,13 @@ private Outputs() { /* no op */ }
         defaultValue) {
       return createNumberSocketHintBuilder(identifier, defaultValue).build();
     }
+
+    public static SocketHint<String> createStringSocketHint(final String identifier,
```

Comment:
The other methods in the class do not have any Javadoc. Should I add for all of them?